### PR TITLE
Project title tooltip

### PR DESCRIPTION
**Pull request by WilliamRichards2017**
Added case description, pedigree, and average coverage to project title dropdown, showing minimum coverage threshold by default, only display min coverage threshold if it is within bounds of the coverage x-axis

### CODE REVIEW

**Comment by AlistairNWard** [comment]
@WilliamRichards2017 Note that in #134, we are only including project description in the project title button

**Comment by WilliamRichards2017** [comment]
Sorry, I'll fix that now and update the request

**Comment by WilliamRichards2017** [comment]
I have removed the pedigree and average coverage from the dropdown. This PR can be tested at 
http://444e0cda.ngrok.io/
I'm assuming that you'll need to be running gene locally on port 4026 though

File: src/components/pages/Navigation.vue [hunk 1]
```diff
@@ -105,22 +104,26 @@ header.theme--dark.v-sheet
...
+              v-model="showCaseMenu"
       >
-      <template v-slot:activator="{ on }">
-        <v-btn text v-on="on" v-if="caseSummary && caseSummary.name">
-          {{ caseSummary.name }}
-        </v-btn>
-      </template>
+        <button v-if="caseSummary && caseSummary.name"
+                offset-y
+        style="margin-bottom: 6px">          {{ caseSummary.name }}</button>
```
Comment (adityaekawade) [inline src/components/pages/Navigation.vue]: Hi @WilliamRichards2017 
There is a blue border that is added when this button is clicked. 
Screen Shot 2020-03-27 at 12 51 47 PM

Can we use the Vuetify button for consistency? 
The following code can fix it: 

```suggestion
          <v-btn v-if="caseSummary && caseSummary.name" text>{{ caseSummary.name }}</v-btn>
```

File: src/components/pages/Navigation.vue [hunk 2]
```diff
@@ -105,22 +104,26 @@ header.theme--dark.v-sheet
...
 
-        <v-card>
-        </v-card>
-      </v-menu>
+        <template slot="popover"
+
+        >
+          <div style="color:black; background:white; width: 245px; text-align: left; margin-right: 5px; font-family: Poppins, sans-serif; font-weight: normal; font-size: 15px">
```
Comment (adityaekawade) [inline src/components/pages/Navigation.vue]: Should we have the same font color as other text on the page and a little smaller font? And maybe wrap the inline css into a class? 

```suggestion
          <div style="color:#6a6a6a; background:white; width: 245px; text-align: left; margin-right: 5px; font-family: Poppins, sans-serif; font-weight: normal; font-size: 14px; padding-right:15px">

```

File: src/components/pages/Navigation.vue [hunk 3]
```diff
@@ -105,22 +104,26 @@ header.theme--dark.v-sheet
...
+            {{caseSummary.description}}
+
+          </div>
+          <div style="height: 20px"></div>
+
+
+          <a v-close-popover style="margin-left: 75px">Close</a>
```
Comment (adityaekawade) [inline src/components/pages/Navigation.vue]: Can we try this with the vuetify button as well? 
```suggestion
          <v-btn text light small color="primary" style="color:rgb(69, 104, 142); margin-left: 50px" v-close-popover >Close</v-btn>
```

**Pull request merged**